build.gradle

```diff
@@ -1,6 +1,16 @@
 subprojects {
   apply from: rootProject.file("gradle/dependencies.gradle")
 
+  File localProperties = project.rootProject.file('local.properties')
```

Comment:
Can this block be in the bintray gradle file or in its own. Just to keep things cleaner here.